app/src/main/AndroidManifest.xml

```diff
@@ -88,19 +88,19 @@
     <activity
       android:name=".app.options.AppLanguageActivity"
       android:label="@string/app_language_activity_title"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/AppTheme.darkMode" />
     <activity
       android:name=".app.options.AudioLanguageActivity"
       android:label="@string/audio_language_activity_title"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/AppTheme.darkMode" />
```

Comment:
We are not aiming to have newer themes here for dark mode. The activities should follow default theme only but the color are needed to be separated for the variations. The changes should only happen in the color files and the concerned layout files unless some special cases. Please go through the wiki for it again and ask any questions if its still not clear.

Reply:
okk , understood , i will try to do it by this way now.